test/testinput/bufr_ncep_sfcshp.yaml

```diff
@@ -0,0 +1,222 @@
+          longName: "Station Pressure"
+          units: "Pa"
+          range: [90000, 105000]
+
+        - name: "ObsValue/presentWeather"
+          coordinates: "longitude latitude"
+          source: variables/presentWeather
+          longName: "Present Weather"
+          units: ""
+
+        - name: "ObsValue/airTemperature"
+          source: variables/airTemperature
+          longName: "Temperature of Air"
+          units: "K"
+          range: [230, 315]
+
+        - name: "ObsValue/dewpointTemperature"
+          coordinates: "longitude latitude"
+          source: variables/dewpointTemperature
+          longName: "Dewpoint Temperature"
+          units: "K"
+          range: [200, 315]
+
+        - name: "MetaData/waterTemperatureMethod"
+          coordinates: "longitude latitude"
+          source: variables/waterTemperatureMethod
+          longName: "Method of Sea Surface Measurement"
+          units: ""
+
+        - name: "ObsValue/seaSurfaceTemperature"
+          coordinates: "longitude latitude"
+          source: variables/sst
+          longName: "Sea Surface Temperature"
+          units: "K"
+          range: [260, 315]
+
+        - name: "ObsValue/seaSurfaceSalinity"
+          coordinates: "longitude latitude"
+          source: variables/sss
+          longName: "Sea Surface Salinity"
+          units: "parts/1000"
+          range: [20, 50]
+
+        - name: "ObsValue/depthBelowWaterSurface"
+          coordinates: "longitude latitude"
+          source: variables/depthBelowWaterSurface
+          longName: "Depth Below Water Surface"
+          units: "m"
+#         range: [-2000, 0]
+
+        - name: "QualityMarker/verticalSignificanceSurfaceObservations"
+          coordinates: "longitude latitude"
+          source: variables/verticalSignificanceSurfaceObservations
+          longName: "Vertical Significance (Surface Observations)"
+          units: ""
+
+        - name: "ObsValue/cloudAmount"
+          coordinates: "longitude latitude"
+          source: variables/cloudAmount
+          longName: "Cloud Amount"
+          units: "1"
+
+        - name: "MetaData/cloudType"
+          coordinates: "longitude latitude"
+          source: variables/cloudType
+          longName: "Cloud Type"
+          units: ""
+
+        - name: "ObsValue/heightOfBaseOfCloud"
+          coordinates: "longitude latitude"
+          source: variables/heightOfBaseOfCloud
+          longName: "Height of Base of Cloud"
+          units: "m"
+
+        - name: "ObsValue/cloudCoverTotal"
+          coordinates: "longitude latitude"
+          source: variables/cloudCoverTotal
+          longName: "Total Cloud Coverage"
+          units: "1"
+
+        - name: "ObsValue/heightAboveSurfaceofBaseofLowestCloud"
+          coordinates: "longitude latitude"
+          source: variables/heightAboveSurfaceofBaseofLowestCloud
+          longName: "Height above Surface of Base of Lowest Cloud"
+          units: ""
+
```

Comment:
units: "" ==> units: "m". please double check if the data value is m, if not, you need to convert.

Reply:
This is code table, no units